### PR TITLE
add symbol table for gc attributes

### PR DESCRIPTION
**Pull request by t-m-z**
Background: in every gpx-download file the attributes are connected to an numeric ID

### CODE REVIEW

**Comment by S-Man42** [comment]
Hm... maybe a copyright issue?

**Comment by t-m-z** [comment]
tricky question - I saw this already realized in Cully Longs book about puzzle caches

Sent from MailDroid

-----Original Message-----

**Comment by S-Man42** [comment]
Maybe we could use the c:geo symbols instead (haven't already checked these licences...), not the same, I know, but maybe less problematic... Or you try to get the permission from GS...

**Comment by t-m-z** [comment]
> Maybe we could use the c:geo symbols instead (haven't already checked these licences...), not the same, I know, but maybe less problematic... Or you try to get the permission from GS...

I checked the website of geocaching.com - there are only limitations to the use of the official logo. So I asked if there limitiations/restrictions using the gc attributes symbols.

I will also check the c:geo-symbols and ask for permission

**Comment by t-m-z** [comment]
> Maybe we could use the c:geo symbols instead (haven't already checked these licences...), not the same, I know, but maybe less problematic... Or you try to get the permission from GS...

Regarding Cully Longs book he wrote as an Acknowledement: ... _images of the geocaching.com website are copyright Groundspeak, Inc. dba Geocaching_. 
I believe there is a way to use these symbols

**Comment by S-Man42** [comment]
I can imagine that the book got an explicit permission. At least you should not forget that we will provide these images as Open Source... That's another dimension. So, a permission for usage would be the best case.

**Comment by t-m-z** [comment]
I am working on it – otherwise I will create similiar icons like c:geo did

 

 

Von: MLorenz42 ***@***.***> 
Gesendet: Wednesday, 5 May 2021 22:19
An: S-Man42/GCWizard ***@***.***>
Cc: Thomas ***@***.***>; Author ***@***.***>
Betreff: Re: [S-Man42/GCWizard] add symbol table for gc attributes (#521)

 

I can imagine that the book got an explicit permission. At least you should not forget that we will provide these images as Open Source... That's another dimension. So, a permission for usage would be the best case.

File: lib/widgets/main_menu/licenses.dart [hunk 1]
```diff
@@ -68,6 +68,7 @@ class LicensesState extends State<Licenses> {
           ['Wikipedia', 'several'],
           ['www.steinerverlag.de', 'Eurythmy'],
           ['www.breitkopf.de', 'Solmisation'],
+          ['game-icons.net\npixabay.com\nwww.clker.com', 'Geocache Attribute'],
```
Comment (S-Man42) [inline lib/widgets/main_menu/licenses.dart:71]: AttributeS

File: assets/symbol_tables/gc_attributes/licences-sources.txt [hunk 2]
```diff
@@ -0,0 +1,79 @@
+game-icons.net => https://creativecommons.org/licenses/by/3.0/
+pixabay.com => kostenlose und lizenzfreie Bilder
```
Comment (S-Man42) [inline assets/symbol_tables/gc_attributes/licences-sources.txt:2]: please in English

File: assets/i18n/en.json [hunk 3]
```diff
@@ -2516,6 +2516,8 @@
   "symboltables_gallifreyan_description": "Language of the Time Lords from the TV show 'Doctor Who'",
   "symboltables_gargish_title": "Gargish",
   "symboltables_gargish_description": "From the video game 'Ultima Online'",
+  "symboltables_gc_attributes_title" : "Geocache attributes",
```
Comment (S-Man42) [inline assets/i18n/en.json:2519]: Capital letters please to be convenient

File: assets/symbol_tables/gc_attributes/licences-sources.txt [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+www.clker.com =>  content is free from any copyrights and trademarks, released under the lastest version of the creative commons CC0 public domain dedication
+
+
+
+ 1 https://game-icons.net/1x1/delapouite/jumping-dog.html#download
+ 2 selfmdae
```
Comment (S-Man42) [inline assets/symbol_tables/gc_attributes/licences-sources.txt:8]: typo

File: lib/widgets/main_menu/licenses.dart (hunk 1 above)
Comment (S-Man42) [reply lib/widgets/main_menu/licenses.dart:71]: I'd add (CC-BY-3.0) and (CC-0)  here.

File: lib/widgets/searchstrings/searchstrings_en.dart [hunk 5]
```diff
@@ -354,6 +354,7 @@ const SEARCHSTRING_EN_SYMBOL_FUTURAMA = '';
 const SEARCHSTRING_EN_SYMBOL_FUTURAMA_2 = '';
 const SEARCHSTRING_EN_SYMBOL_GALLIFREYAN = 'gallifreyan';
 const SEARCHSTRING_EN_SYMBOL_GARGISH = 'gargish';
+const SEARCHSTRING_EN_SYMBOL_GC_ATTRIBUTES = '';
```
Comment (S-Man42) [inline lib/widgets/searchstrings/searchstrings_en.dart:357]: attributes

File: lib/widgets/searchstrings/searchstrings_common.dart [hunk 6]
```diff
@@ -350,6 +350,7 @@ const SEARCHSTRING_COMMON_SYMBOL_FUTURAMA = 'futurama mattgroening';
 const SEARCHSTRING_COMMON_SYMBOL_FUTURAMA_2 = 'futurama mattgroening';
 const SEARCHSTRING_COMMON_SYMBOL_GALLIFREYAN = 'gallifrey doctorwho timelords drwho';
 const SEARCHSTRING_COMMON_SYMBOL_GARGISH = 'ultimaonline herman miller';
+const SEARCHSTRING_COMMON_SYMBOL_GC_ATTRIBUTES = '';
```
Comment (S-Man42) [inline lib/widgets/searchstrings/searchstrings_common.dart:353]: geocaching geocache listing

File: lib/widgets/main_menu/licenses.dart (hunk 1 above)
Comment (t-m-z) [reply lib/widgets/main_menu/licenses.dart:71]: Done

File: assets/symbol_tables/gc_attributes/licences-sources.txt (hunk 2 above)
Comment (t-m-z) [reply assets/symbol_tables/gc_attributes/licences-sources.txt:2]: done

File: assets/i18n/en.json (hunk 3 above)
Comment (t-m-z) [reply assets/i18n/en.json:2519]: done

File: lib/widgets/searchstrings/searchstrings_en.dart (hunk 5 above)
Comment (t-m-z) [reply lib/widgets/searchstrings/searchstrings_en.dart:357]: done

File: lib/widgets/searchstrings/searchstrings_common.dart (hunk 6 above)
Comment (t-m-z) [reply lib/widgets/searchstrings/searchstrings_common.dart:353]: done

**Comment by t-m-z** [comment]
PR done

**Pull request merged**